NodeJS-HW3/src/db/sequelize.ts

```diff
@@ -0,0 +1,7 @@
+import { Sequelize } from 'sequelize';
+
+const sequelize = new Sequelize(
+  'postgres://postgres:jack2000@localhost:5432/users',
```

Comment:
it is better to retrieve db credentials from .env file (dotenv package), and never store actual values in version control